src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -272,7 +272,7 @@
      :amount-out                (:amount-out new-path)
      :approval-contract-address (:approval-contract-address new-path)
      :approval-required         (:approval-required new-path)
-     :estimated-time            (:estimated-time new-path)
+     :estimated-time            (:tx-estimated-time new-path)
```

Comment:
thanks for spotting this!

Reply:
I think I'll revert this commit and add time estimation changes with a separate PR. Seems like time estimations is shown in seconds now, so might need more than just this.